hedera-mirror-importer/src/test/java/com/hedera/mirror/importer/util/UtilityTest.java

```diff
@@ -21,20 +21,42 @@
  */
 
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
 import static org.junit.jupiter.api.Assertions.*;
 
 import com.google.protobuf.ByteString;
+import com.google.protobuf.InvalidProtocolBufferException;
 import com.hederahashgraph.api.proto.java.AccountID;
 import com.hederahashgraph.api.proto.java.ContractLoginfo;
+import com.hederahashgraph.api.proto.java.Key;
 import com.hederahashgraph.api.proto.java.Timestamp;
 import com.hederahashgraph.api.proto.java.TransactionID;
 import java.time.Instant;
+import org.bouncycastle.util.encoders.Hex;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.CsvSource;
 
-class UtilityTest {
+import com.hedera.mirror.importer.exception.ParserException;
+
+public class UtilityTest {
```

Comment:
You/IDE added a public on the class
```suggestion
class UtilityTest {
```

Reply:
I reuse a static variable inside this class in other tests, hence the change to public.